scripts/refresh-dashboard.sh

```
@@ -0,0 +1,237 @@
+while IFS='|' read -r repo num title branch checks; do
+  [[ -z "$repo" ]] && continue
+  # Determine sprite from branch prefix
+  sprite=""
+  case "$branch" in
+    thorn/*) sprite="Thorn" ;;
+    bramble/*) sprite="Bramble" ;;
+    willow/*) sprite="Willow" ;;
+    fern/*) sprite="Fern" ;;
+    moss/*) sprite="Moss" ;;
+    claw/*) sprite="Claw" ;;
+    *) sprite="—" ;;
```

Shellcheck (0.11.0)

[warning] sprite appears unused. Verify use (or export if used externally).

(SC2034)